.github/workflows/clean-repo.yml

```diff
@@ -0,0 +1,40 @@
+name: "clean repo"
+
+on:
+  schedule:
+    - cron: "0 1 1 * *" # Runs at 01:00, on day 1 of the month
```

Comment:
Should we pick a different day so this runs on a different day than the What's new action?

Reply:
The .NET Version Sweeper was the first tool in our repo to be setup as a CRON job. I bumped it up an hour from that time. I think the What's new tool should be bumped another hour.